server/simple_server.go

```diff
@@ -306,16 +304,15 @@ func (s *SimpleServer) Register(svcI Service) error {
 		// register all context endpoints with our wrapper
 		for path, epMethods := range mcs.JSONEndpoints() {
 			for method, ep := range epMethods {
-				endpointName := metricName(prefix, path, method)
+				fullPath := strings.TrimPrefix(prefix+path, "/")
```

Comment:
Yeah, you're right, the TrimPrefix is only used for the metrics and not the handler so I'll move that into metrics.go.